docs/02_tokenization_files/figure-html/ngramtokens-1.svg

```diff
@@ -39,630 +39,630 @@
 <polyline points='158.76,279.72 158.76,39.07 ' style='stroke-width: 0.53; stroke: #DEDEDE; stroke-linecap: butt;' />
 <polyline points='246.65,279.72 246.65,39.07 ' style='stroke-width: 0.53; stroke: #DEDEDE; stroke-linecap: butt;' />
 <polyline points='334.55,279.72 334.55,39.07 ' style='stroke-width: 0.53; stroke: #DEDEDE; stroke-linecap: butt;' />
-<circle cx='81.95' cy='77.11' r='1.95' style='stroke-width: 0.71; stroke: #6EC95D; stroke-opacity: 0.80; fill: #6EC95D; fill-opacity: 0.80;' />
```

Comment:
These diffs are due to the jitter; we might want to [set a seed](https://stackoverflow.com/a/62692568/5468471).

Reply:
Done!